docs/ref/contrib/admin/index.txt

```diff
@@ -1287,6 +1287,22 @@ subclass::
     a full count on the table which can be expensive if the table contains a
     large number of rows.
 
+.. attribute:: ModelAdmin.sortable_by
+
+    .. versionadded:: 2.1
+
+    By default, the change list page allows users to interactively sort it by
+    all model fields (and callables that have ``admin_order_field`` property
+    set) specified by :attr:`list_display`. However, you might want to disable
+    interactive sorting for any/all of them.
+
+    With ``sortable_by`` you can specify a collection of a subset of these
+    items. You can even disable customization of such ordering by specifying an
+    empty collection.
```

Comment:
Not 100% against the wording here but I find `...you can specify a collection of a subset of ...` difficult. 
(It doesn't quite read for me.) 

Taking inspiration from `search_fields` (above) maybe something like:

```
With ``sortable_by`` you can specify a subset of these items.
Set ``sortable_by`` to a collection of field names that will be available to sort by.
You can disable customization of such ordering by specifying an
empty collection.
```

Reply:
English isn't my mother tongue and I admit my style can be really baroque :smile: . So feel free to simplify the wording as much as you like. 

One fact we might need to want to make clear is that the items one can use in this settings are not necessarily always model field names, the can also be model or ModelAdmin methods, or functions. Just like in `list_display`.

Reply:
Yes good point. Perhaps `...collection of column names...` would be enough. (It already mentions the _other callables_ aspect in the preceding paragraph.)